src/apis/staked-relayer.ts

```diff
@@ -24,6 +24,12 @@ export interface StakedRelayerAPI {
     getAllActiveStatusUpdates(): Promise<Array<{ id: u256; statusUpdate: StatusUpdate }>>;
     getAllInactiveStatusUpdates(): Promise<Array<{ id: u256; statusUpdate: StatusUpdate }>>;
     getAllStatusUpdates(): Promise<Array<{ id: u256; statusUpdate: StatusUpdate }>>;
+    getFees(stakedRelayerId: AccountId): Promise<PolkaBTC>;
+    getSLA(stakedRelayerId: AccountId): Promise<number>;
```

Comment:
SLA and score is the same thing in the current model. So we only need one of the two. All the scoring additions will be handled in the parachain since we need to have them onchain for fee payouts.

Reply:
I followed the fee model document ![Screenshot from 2020-12-10 17-01-50](https://user-images.githubusercontent.com/23065004/101804431-7b0b1380-3b09-11eb-92dd-3f22c259ada4.png)

Reply:
Yes, but that is all hidden in the UI. My bad on saying they are the same. However, the score and reward function will not be exposed externally. This is only for the internal computations in the parachain. In the UI and lib, all we need is the fees earned by the staked relayer and the SLA score.